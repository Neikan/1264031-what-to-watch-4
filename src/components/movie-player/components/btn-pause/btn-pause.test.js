import React from "react";
import renderer from "react-test-renderer";
import BtnPause from "./btn-pause";


describe(`Test BtnPause component`, () => {
  test(`BtnPause component is created and rendered correctly`, () => {
    const tree = renderer.create(
        <BtnPause />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
