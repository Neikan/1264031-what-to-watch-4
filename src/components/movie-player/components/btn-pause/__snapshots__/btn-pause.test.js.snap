// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test BtnPause component BtnPause component is created and rendered correctly 1`] = `
Array [
  <svg
    height="21"
    viewBox="0 0 14 21"
    width="14"
  >
    <use
      xlinkHref="#pause"
    />
  </svg>,
  <span>
    Pause
  </span>,
]
`;
